### ISSUE TITLE
no demo page on github

### ISSUE THREAD

**Issue by yairEO** [agence-webup/dropmic#2]
y u no demo page?

https://pages.github.com/

**Comment by TixieSalander** [comment]
Yep you're right, I should do that so people can test the plugin without downloading it.

**Comment by TixieSalander** [comment]
[_one eternity later_] Yeah I'm sorry i didn't got time to work on that, but if anyone have an idea to make something (necessarily elaborate) your welcome to suggest something in a PR. That would be very welcome! ❤️
